### PR TITLE
[SEMI-MODULAR] [READY] Underwear Ports, in 2022? Also fixing the Dutch jacket hiding your hands

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Ports some undies from https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pulls

<details>
<summary> The Contents </summary>

image
Leotards. Yes, this is a closet cosplay of Panam Palmer from Cyberpunk 2077. Cope.

image
Stirrup socks. These come in short, normal-length, knee-high, thigh-high, pantyhose, leggings, and striped knee/thigh-high. Yes, this is a closet cosplay of Envy from Fullmetal Alchemist. Yes, they were my favorite Homunculus.

image
image
image
Turtleneck undershirt. 

image
image
image
Comes without sleeves too.

image
Easy to recolor.

image
image
Off-shoulder top, with brastrap.

image
image
Off-shoulder top, without brastrap. Just put bra, bikini under there.

image
Recolorable fishnets.

image
image
These don't have the obligatory bra cups that the old ones did. Yes, I made a Security pinup guy. He's pretty cool.

image
Sarashi chest bandages. For the samurai/sukeban/fem Goro Majima in you.

image
Leggings.

image
Binders. There's a strapless version.

</details>

## How This Contributes To The Skyrat Roleplay Experience
Customization good.

## Changelog

:cl:
add: Adds some binders, turtlenecks, recolorable fishnet attire, off-shoulder tops, leotards, leggings, sarashi, and stirrup socks to character setup.
add: Makes thongs recolorable.
fix: Makes the Dutch Jacket have hand cutouts again.
del: Removes the alt strapless bra, it doesn't work and the sprite for it doesn't exist. 
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
Update: This also fixes the Dutch Jacket having no hands.

**Comment by Gandalf2k15** [comment]
Sorry bot says I gotta close your pr

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -549,36 +612,54 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	gender = FEMALE
 	use_static = null
 
-/datum/sprite_accessory/undershirt/bra_strapless_alt
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm:552]: ?

Comment (Nerev4r) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm:552]: Oh, forgot to include this in the changelog. The sprite doesn't physically exist so this doesn't work, I'm not sure if it ever did.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -549,36 +612,54 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
-	name = "Fishnet - sleeves"
+	name = "Fishnet - Sleeves"
 	icon_state = "fishnet_sleeves"
 	gender = FEMALE
 	use_static = TRUE
 
+/datum/sprite_accessory/undershirt/fishnet_sleeves/alt
+	name = "Fishnet - Sleeves Alt"
+	icon_state = "fishnet_sleeves_alt"
+	gender = FEMALE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 3]
```diff
@@ -549,36 +612,54 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
-	name = "Fishnet - gloves"
+	name = "Fishnet - Gloves"
 	icon_state = "fishnet_gloves"
 	gender = FEMALE
 	use_static = TRUE
 
+/datum/sprite_accessory/undershirt/fishnet_gloves/alt
+	name = "Fishnet - Gloves Alt"
+	icon_state = "fishnet_gloves_alt"
+	gender = FEMALE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 4]
```diff
@@ -549,36 +612,54 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
-	name = "Fishnet - top"
+	name = "Fishnet - Top"
 	icon_state = "fishnet_body"
 	gender = FEMALE
 	use_static = TRUE
 
+/datum/sprite_accessory/undershirt/fishnet_base/alt
+	name = "Fishnet - Top Alt"
+	icon_state = "fishnet_body_alt"
+	gender = FEMALE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 5]
```diff
@@ -169,6 +169,11 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	icon_state = "socks_knee"
 	use_static = null
 
+/datum/sprite_accessory/socks/socks_knee/stirrups
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
/datum/sprite_accessory/socks/stirrups_knee
```
Don't see a reason for this to be subtype at a glance

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 6]
```diff
@@ -184,6 +189,16 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	icon_state = "socks_norm"
 	use_static = null
 
+/datum/sprite_accessory/socks/socks_norm/stirrups
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
/datum/sprite_accessory/socks/stirrups_norm
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 7]
```diff
@@ -184,6 +189,16 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
+	name = "Normal Stirrups"
+	icon_state = "socks_norm-stir"
+	use_static = null
+
+/datum/sprite_accessory/socks/socks_mid_stirrups
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
/datum/sprite_accessory/socks/stirrups_mid
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 8]
```diff
@@ -194,11 +209,21 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	icon_state = "socks_thigh"
 	use_static = null
 
+/datum/sprite_accessory/socks/socks_thigh/stirrups
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
/datum/sprite_accessory/socks/stirrups_thigh
```
Etc

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 9]
```diff
@@ -298,6 +356,12 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
 	gender = FEMALE
 	use_static = TRUE
 
+/datum/sprite_accessory/underwear/fishnet_lower/alt
+	name = "Panties - Fishnet Alt"
+	icon_state = "fishnet_lower_alt"
+	gender = FEMALE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 10]
```diff
@@ -632,3 +713,56 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
+	name = "Shirt - Black Stripes"
+	icon_state = "longstripe"
+	gender = NEUTER
+	use_static = TRUE
+
+/datum/sprite_accessory/undershirt/striped/blue
+	name = "Shirt - Blue Stripes"
+	icon_state = "longstripe_blue"
+	gender = NEUTER
+	use_static = TRUE
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 11]
```diff
@@ -632,3 +713,56 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
+	name = "Binder"
+	icon_state = "binder"
+	gender = MALE
+	use_static = null
+
+/datum/sprite_accessory/undershirt/binder/strapless
+	name = "Binder - Strapless"
+	icon_state = "binder_strapless"
+	gender = MALE
+	use_static = null
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 12]
```diff
@@ -632,3 +713,56 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
+	name = "Sweater - Turtleneck"
+	icon_state = "turtleneck"
+	use_static = null
+	gender = NEUTER
+
+/datum/sprite_accessory/undershirt/turtleneck/sleeveless
+	name = "Sweater - Sleeveless Turtleneck"
+	icon_state = "turtleneck_sleeveless"
+	gender = NEUTER
+	use_static = null
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 13]
```diff
@@ -632,3 +713,56 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
+	name = "Shirt - Leotard"
+	icon_state = "leotard"
+	gender = FEMALE
+	use_static = null
+
+/datum/sprite_accessory/undershirt/leotard/turtleneck
+	name = "Shirt - Turtleneck Leotard"
+	icon_state = "leotard_turtleneck"
+	gender = FEMALE
+	use_static = null
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

**Review by tf-4 — CHANGES_REQUESTED** [review]
Mostly just redundant subtype stuff

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm [hunk 14]
```diff
@@ -549,36 +611,51 @@ GLOBAL_LIST_EMPTY(cached_mutant_icon_files)
...
-
 /datum/sprite_accessory/undershirt/striped_bra
 	name = "Bra - Striped"
 	icon_state = "striped_bra"
 	gender = FEMALE
 	use_static = null
 
+/datum/sprite_accessory/undershirt/sarashi
+	name = "Bra - Sarashi"
+	icon_state = "bandages"
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm:622]: this the correct iconstate?

**Review by Zonespace27 — DISMISSED** [review]

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm (hunk 14 above)
Comment (Nerev4r) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories.dm:622]: Yes.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Most of these have nice shapes, but they severely lack in shading. Look at the first few, you need to squint to see the details on most of them, because there's barely any difference in color.

The last two, the binder and the leggings, are also incredibly flat and/or weirdly shaded.

The chest wrap looks fine for the most part. The fishnets, well, it's fishnet, you won't really get much better than that.

Overall, I feel like a few of these could just use a bit more shading, and more depth in their color, because having to squint to see color differences isn't too great.

Also, for the dutch jacket, I'd just honestly recommend you put it in a different PR, it'd get merged much faster.

**Comment by Nerev4r** [comment]
image
image
image
Regular
image
image
image
Sleeveless
Leotard update.

image
image
image
Regular
image
image
image
Sleeveless
Turtleneck update.


image
image
image
image
Off-shoulder top, with brastrap.

image
Off-shoulder top, without brastrap. Just put bra, bikini under there if you want it.

image
image
image
Leggings. I should note these are counted as socks.

image
image
image
Binders. Happy pride month.

image
image
image
Strapless binders.

**Comment by GoldenAlpharex** [comment]
> image image image Regular image image image Sleeveless Leotard update.
> 
> image image image Regular image image image Sleeveless Turtleneck update.
> 
> image image image image Off-shoulder top, with brastrap.
> 
> image Off-shoulder top, without brastrap. Just put bra, bikini under there if you want it.
> 
> image image image Leggings. I should note these are counted as socks.
> 
> image image image Binders. Happy pride month.
> 
> image image image Strapless binders.

Oh yeah, that's much, much better. Nice work @CandleJaxx (and whoever else worked on these, if anyone else did, good job to you too).

That being said, are the binders meant to look like your nipples are showing through? Feels a bit odd compared to the other bras to me, but that's my only critique for all of those sprites.

**Comment by Nerev4r** [comment]
handled.

**Review by GoldenAlpharex — APPROVED** [review]
Looks fine to me, ideally we'd have uniform greyscale colors for all of these, but I don't want to deal with this for any longer.

**Pull request merged**